client/src/pages/admin/users.tsx

```diff
@@ -79,7 +79,7 @@ function Page() {
 export default function () {
   return (
     <ActiveCourseProvider>
-      <SessionProvider allowedRoles={[CourseRole.Manager]}>
+      <SessionProvider allowedRoles={[CourseRole.Manager]} anyCoursePowerUser={true}>
```

Comment:
just in case: you may omit `true` and just have attribute for truthy values

Reply:
Thank you, fixed.